app/_landing_pages/gateway-manager.yaml

```
@@ -1,10 +1,184 @@
+                style A stroke:none,fill:#0E44A2,color:#fff
+                style B stroke:none,fill:#0E44A2,color:#fff
+
+  
+  - header:
+      type: h2
+      text: "Cloud Hosted"
+    columns:
+      - blocks:
+          - type: card
+            config:
+              title: Serverless Gateways
+              description: |
+                Serverless Gateways provide fully managed data plane nodes with automatic scaling.
+                Ideal for rapid provisioning and API experimentation.
+              icon: /assets/icons/menu.svg
+              cta:
+                text: Learn more
+                url: /serverless-gateways/
+      - blocks:
+          - type: card
+            config:
+              title: Dedicated Cloud Gateways
+              description: |
+                Dedicated Cloud Gateways allow full control over infrastructure placement while
+                providing fully managed data plane nodes.
+              icon: /assets/icons/gateway.svg
+              cta:
+                text: Learn more
+                url: /dedicated-cloud-gateways/
+
+  - header:
+      type: h2
+      text: "Configurable Entities"
+  - columns:
+      - blocks:
+          - type: entity_card
+            config:
+              entity: service
+      - blocks:
+          - type: entity_card
+            config:
+              entity: route
+      - blocks:
+          - type: entity_card
+            config:
+              entity: consumer
+      - blocks:
+          - type: entity_card
+            config:
+              entity: consumer-group
+  - columns:
+      - blocks:
+          - type: entity_card
+            config:
+              entity: upstream
+      - blocks:
+          - type: entity_card
+            config:
+              entity: target
+      - blocks:
+          - type: entity_card
+            config:
+              entity: plugin
+      - blocks:
+          - type: entity_card
+            config:
+              entity: certificate
+      - blocks:
+          - type: entity_card
+            config:
+              entity: vault
+  - header:
+      type: h2
+      text: "Learn More"
+    columns:
+      - blocks:
+          - type: card
+            config:
+              title: Control Plane Groups
+              description: |
+                Control Plane Groups enable teams to share infrastructure while maintaining
+                independent configurations.
+              icon: /assets/icons/gateway.svg
+              cta:
+                text: Learn more
+                url: /gateway-manager/control-plane-groups/
+      - blocks:
+          - type: card
+            config:
+              title: Data Plane Nodes
+              description: |
+                Learn how to set up and manage self-managed and cloud-hosted data plane nodes.
+              icon: /assets/icons/gateway.svg
+              cta:
+                text: Learn more
+                url: /gateway-manager/data-plane-nodes/
```

```suggestion
                url: /gateway-manager/data-plane-nodes/
           - blocks:
          - type: card
            config:
              title: {{site.base_gateway}} control plane and data plane communication
              description: |
                Learn about how control planes and data plane nodes communicate with each other.
              icon: /assets/icons/gateway.svg
              cta:
                text: Learn more
                url: /gateway/cp-dp-communication/
```
The spacing is probably off, but I thought this could be a good spot to link to this.